lib/_tls_common.js

```diff
@@ -34,8 +34,7 @@ const { SSL_OP_CIPHER_SERVER_PREFERENCE } = process.binding('constants').crypto;
 // Lazily loaded
 var crypto = null;
 
-const binding = process.binding('crypto');
-const NativeSecureContext = binding.SecureContext;
+const NativeSecureContext = process.binding('crypto').SecureContext;
```

Comment:
I was unsure if that should be changed or not because of the different names. I'm happy to change this though if this is the preferred syntax. Do you know if it is? Thx

Reply:
That still keeps the name to `NativeSecureContext` so I _think_ it's fine.